### PR TITLE
V2 Model Compression experiment result

### PR DESCRIPTION
**Pull request by Fiascolsy**
### Description ###
Some simple tests to measure the performance of v2 pruner.

### Checklist ###
  - [ ] test case
  - [ ] doc

### CODE REVIEW

File: docs/en_US/Compression/v2_pruning_algo.rst [hunk 1]
```diff
@@ -55,6 +55,29 @@ User configuration for Level Pruner
 
 .. autoclass:: nni.algorithms.compression.v2.pytorch.pruning.LevelPruner
 
+Performance Test
```
Comment (QuanluZhang) [inline docs/en_US/Compression/v2_pruning_algo.rst:58]: all these performance numbers are obtained from our example code?

Comment (Fiascolsy) [reply docs/en_US/Compression/v2_pruning_algo.rst:58]: Yes, the seed was set,  and some hyperparameters may change.

Comment (QuanluZhang) [reply docs/en_US/Compression/v2_pruning_algo.rst:58]: is it better to also provide the hyper-parameters that you use?

File: docs/en_US/Compression/v2_pruning_algo.rst [hunk 2]
```diff
@@ -55,6 +55,29 @@ User configuration for Level Pruner
...
+     - Sparsity
+   * - VGG-16
+     - 93.86%
+     - 14.98M
+     - 313.46M
+     - 
+   * - Pruned VGG-16
+     - 93.74%
+     - 14.98M
+     - 313.46M
```
Comment (QuanluZhang) [inline docs/en_US/Compression/v2_pruning_algo.rst:78]: why parameters and flops do not change in the pruned model?

File: docs/en_US/Compression/v2_pruning_algo.rst [hunk 3]
```diff
@@ -200,6 +309,29 @@ User configuration for Activation APoZ Rank Pruner
...
+     - FLOPs
+     - Sparsity
+   * - VGG-16
+     - 93.86%
+     - 14.98M
+     - 313.46M
+     - 
+   * - Pruned VGG-16
+     - 87.24%
+     - 0.64M
```
Comment (QuanluZhang) [inline docs/en_US/Compression/v2_pruning_algo.rst:331]: why parameters are reduce so much with 80% sparsity?

File: docs/en_US/Compression/v2_pruning_algo.rst [hunk 4]
```diff
@@ -401,6 +579,29 @@ User configuration for AGP Pruner
...
+     - Sparsity
+   * - VGG-16
+     - 93.86%
+     - 14.98M
+     - 313.46M
+     - 
+   * - Pruned VGG-16
+     - 76.94%
+     - 
+     -
```
Comment (QuanluZhang) [inline docs/en_US/Compression/v2_pruning_algo.rst:602]: why the two numbers are missing?

File: docs/en_US/Compression/v2_pruning_algo.rst [hunk 5]
```diff
@@ -478,6 +702,29 @@ User configuration for Simulated Annealing Pruner
...
+     - Sparsity
+   * - VGG-16
+     - 93.86%
+     - 14.98M
+     - 313.46M
+     - 
+   * - Pruned VGG-16
+     - 33.6% (wo FT)
+     - 
+     -
```
Comment (QuanluZhang) [inline docs/en_US/Compression/v2_pruning_algo.rst:725]: what is the meaning of "wo FT", and there are also two missing numbers

File: docs/en_US/Compression/v2_pruning_algo.rst (hunk 1 above)
Comment (Fiascolsy) [reply docs/en_US/Compression/v2_pruning_algo.rst:58]: Now this has been provided.

File: docs/en_US/Compression/v2_pruning_algo.rst (hunk 2 above)
Comment (Fiascolsy) [reply docs/en_US/Compression/v2_pruning_algo.rst:81]: This is fine-grained algorithm, it has no speedup procedure.

File: docs/en_US/Compression/v2_pruning_algo.rst (hunk 3 above)
Comment (Fiascolsy) [reply docs/en_US/Compression/v2_pruning_algo.rst:349]: The pruning behavior may differ according to various default `mode` setting. It has three types: 'normal', 'global' and 'dependency_aware'. In the intermediate `config_list` we can see it does prune nearly 0.8 in each layer.

File: docs/en_US/Compression/v2_pruning_algo.rst (hunk 4 above)
Comment (Fiascolsy) [reply docs/en_US/Compression/v2_pruning_algo.rst]: Because the result was not saved at that time, now it has been added.

File: docs/en_US/Compression/v2_pruning_algo.rst (hunk 5 above)
Comment (Fiascolsy) [reply docs/en_US/Compression/v2_pruning_algo.rst]: At first I want the result 'without finetune', but this time it has been normal.
Just like the workflow: `prune -> speedup -> finetune`

**Review by QuanluZhang — APPROVED** [review]

**Review by J-shang — APPROVED** [review]

**Comment by J-shang** [comment]
These results have already been updated. But the result has little difference, I think maybe we need a more significant benchmark...
